src/CoreGraphics/CGAffineTransform.cs

```diff
@@ -106,10 +106,30 @@ public void Multiply (CGAffineTransform b)
 			x0 = a.x0 * b.xx + a.y0 * b.xy + b.x0;
 			y0 = a.x0 * b.yx + a.y0 * b.yy + b.y0;
 		}
+
+		public enum MatrixOrder {
```

Comment:
Do we want this enum to be a nested enum in `CGAffineTransform`, or a normal enum in CoreGraphics?

IMHO this sounds like an enum that could end up being used elsewhere, in which case a non-nested enum would make more sense.

@spouliot what do you think?

Reply:
hmm... bad timing. This refresh after I merged

Reply:
@spouliot should I make a new PR with that change?

Reply:
@VincentDondain yes, even if it does not have an immediate impact, it might prove useful to other API in the future (and it would be a breaking change to move it then).